### ISSUE TITLE
Crash when switching from Main to Background Player on long video

### ISSUE THREAD

**Issue by Grammost** [TeamNewPipe/NewPipe#1779]
- [x] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.
- [x] I checked if the issue/feature exists in the latest version.
- [x] I did use the [incredible bugreport to markdown converter](https://teamnewpipe.github.io/CrashReportToMarkdown/) to paste bug reports.

## Steps to reproduce
1. Open this video: https://www.youtube.com/watch?v=SxNcKXjfaQo
2. Open in regular player.
3. Switch to Background player.
4. Then the app freezes and crashes for me, see below.
![mp4_20181002_160023_2](https://user-images.githubusercontent.com/34781145/46353806-5aa7ba80-c65d-11e8-9210-a7a507a031b6.gif)

## Exception
* __User Action:__ ui error
* __Request:__ App crash, UI failure
* __Content Language:__ US
* __Service:__ none
* __Version:__ 0.14.1
* __OS:__ Linux Android 8.1.0 - 27

```
java.lang.NullPointerException: Attempt to invoke virtual method 'long com.google.android.exoplayer2.SimpleExoPlayer.getCurrentPosition()' on a null object reference
	at org.schabi.newpipe.player.BasePlayer.seekBy(BasePlayer.java:943)
	at org.schabi.newpipe.player.BasePlayer.onFastForward(BasePlayer.java:894)
	at org.schabi.newpipe.player.VideoPlayer.onFastForward(VideoPlayer.java:585)
	at org.schabi.newpipe.player.MainVideoPlayer$PlayerGestureListener.onDoubleTap(MainVideoPlayer.java:959)
	at android.view.GestureDetector.onTouchEvent(GestureDetector.java:565)
	at org.schabi.newpipe.player.MainVideoPlayer$PlayerGestureListener.onTouch(MainVideoPlayer.java:1086)
	at android.view.View.dispatchTouchEvent(View.java:11772)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2962)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2643)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2968)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2600)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2968)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2600)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2968)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2600)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2968)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2600)
	at com.android.internal.policy.DecorView.superDispatchTouchEvent(DecorView.java:448)
	at com.android.internal.policy.PhoneWindow.superDispatchTouchEvent(PhoneWindow.java:1829)
	at android.app.Activity.dispatchTouchEvent(Activity.java:3307)
	at android.support.v7.view.WindowCallbackWrapper.dispatchTouchEvent(WindowCallbackWrapper.java:68)
	at com.android.internal.policy.DecorView.dispatchTouchEvent(DecorView.java:410)
	at android.view.View.dispatchPointerEvent(View.java:12015)
	at android.view.ViewRootImpl$ViewPostImeInputStage.processPointerEvent(ViewRootImpl.java:4795)
	at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:4609)
	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4147)
	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:4200)
	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:4166)
	at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:4293)
	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:4174)
	at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:4350)
	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4147)
	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:4200)
	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:4166)
	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:4174)
	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4147)
	at android.view.ViewRootImpl.deliverInputEvent(ViewRootImpl.java:6661)
	at android.view.ViewRootImpl.doProcessInputEvents(ViewRootImpl.java:6635)
	at android.view.ViewRootImpl.enqueueInputEvent(ViewRootImpl.java:6596)
	at android.view.ViewRootImpl$WindowInputEventReceiver.onInputEvent(ViewRootImpl.java:6764)
	at android.view.InputEventReceiver.dispatchInputEvent(InputEventReceiver.java:186)
	at android.os.MessageQueue.nativePollOnce(Native Method)
	at android.os.MessageQueue.next(MessageQueue.java:325)
	at android.os.Looper.loop(Looper.java:142)
	at android.app.ActivityThread.main(ActivityThread.java:6494)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:440)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)

-------------------

```

**Comment by theScrabi** [comment]
I don't have this issue with my android 7, maybe this is another android 8 issue.

**Comment by Kowalski7** [comment]
I have Android 7.1.1 and this happens to me too, and not only with Background Player, but with the Popup Player too.

**Comment by TobiGr** [comment]
@Kowalski7 I guess the bug you are facing is something different and only happens with NewPipe 0.14.2. It already fixed (see #1874).

**Comment by reedknight** [comment]
This problem is occuring in version 0.18.0 on android 8.1.0. App crashes on SOME videos when switching to background or popup modes. I think this is happening only with music videos.
